torch/_inductor/codegen/common.py

```diff
@@ -331,28 +337,33 @@ def cpp_argdefs(self):
 
         call_args = []
         arg_defs = []
+        arg_types = []
         for inplaced in unique(self.inplace_buffers.values()):
             outer = inplaced.other_names[-1]
             inner = inplaced.inner_name
             dtype = buffer_types[outer]
             arg_defs.append(f"{DTYPE_TO_CPP[dtype]}* __restrict__ {inner}")
-            call_args.append(f"c_void_p({outer}.data_ptr())")
+            call_args.append(self.wrap_ptr_arg(outer, dtype))
+            arg_types.append(f"{DTYPE_TO_CPP[dtype]}*")
```

Comment:
Could use some refactoring here by reusing `DTYPE_TO_CPP[dtype]`.

Reply:
Thanks for the suggestion. I've refactored the code to remove the duplicated `DTYPE_TO_CPP[dtype]`.